software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -142,6 +140,27 @@
     MethodEffector<Void> LEAVE_RIAK_CLUSTER = new MethodEffector<Void>(RiakNode.class, "leaveCluster");
     MethodEffector<Void> COMMIT_RIAK_CLUSTER = new MethodEffector<Void>(RiakNode.class, "commitCluster");
 
+    AttributeSensor<String> NODE_GET_FSM_TIME_MEAN = Sensors.newStringSensor("node_get_fsm_time_mean", "Time between reception of client read request and subsequent response to client");
```

Comment:
Sensor names should be of the form `riak.node.fsm_time.mean` and should indicate in the description what the units are (milliseconds, perhaps?)